web/src/pages/suppRequests/ApproveApplication.tsx

```diff
@@ -0,0 +1,72 @@
+// (c) Tecnologico de Monterrey 2022, rights reserved.
+
+import RequestCard from "../../components/RequestCard/RequestCard";
+import back from "../../assets/background/login.png";
+import { useLazyLoadQuery } from "react-relay";
+import graphql from "babel-plugin-relay/macro";
+
+import { ApproveApplicationQuery, ApproveApplicationQuery$data } from "./__generated__/ApproveApplicationQuery.graphql";
+
+const ApproveApplication = (): JSX.Element => {
+  const data: ApproveApplicationQuery$data = useLazyLoadQuery<ApproveApplicationQuery>(
+    graphql`
+      query ApproveApplicationQuery($application_status_id: ID!) {
+        applicationByStatusID(application_status_id: $application_status_id) {
+          title
+          id
+          user {
+            id
+            name
+            first_lastname
+            second_lastname
+          }
+          citation {
+            id
+            title
+          }
+        }
+      }
+    `,
+    { application_status_id: "" },
+  );
+
+  const exampleLabels = [{ label: "Cultura" }, { label: "Baile" }];
```

Comment:
If query arguments can be sent null modify schema and handle null values inside query resolver.